src/screens/IndexScreen.js

```diff
@@ -195,18 +223,17 @@ export const IndexScreen = ({ navigation, route }) => {
 
   return (
     <SafeAreaViewFlex>
-      {query === QUERY_TYPES.POINTS_OF_INTEREST ? (
+      {query === QUERY_TYPES.POINTS_OF_INTEREST && (
```

Comment:
Why did you replace the `?` operator with the `&&` operator?

Reply:
I replaced it because of our more consistent way of rendering "something or nothing". I felt it like a interruption of our code flow and way of reading it. why do we need a short-hand if-else here with explicitly rendering `null` if the query is not for POIs? when reading the code, we want to do something if we have a query for POIs and nothing else. it is like rendering other stuff only if some condition is true. imagine our `DataListSection` component with `... ? ... : null` all the time:

<img width="660" alt="Bildschirmfoto 2022-07-25 um 09 49 30" src="https://user-images.githubusercontent.com/1942953/180726253-29160316-090e-40f2-a676-e3828a03c39d.png">

this would become a lot more unreadable and confusing in my opinion. that's why i adjusted it here as well when i saw it.

Reply:
Yes, it certainly reads better.